### ISSUE TITLE
Two Feature Requests

### ISSUE THREAD

**Issue by mark-hahn** [dstepanov/coffeescript-netbeans#9]
I know both of these are things that netbeans should provide, but I am hoping you could figure out how to get them to work somehow.

1) I need line numbers in compiled output that displays in the netbeans "Compiling XXX"  window .  When I get an error in node I would really like to link it to your output but the lack of line numbers makes that impossible.  Maybe as an option you could put  something like  `/* 0012 */` at the beginning of every output line.

2) Provide a shortcut key to compile the current page and produce this window.

**Comment by dstepanov** [comment]
1) Why couldn't you just open generated js file?
Output only displays output of compiler

**Comment by nelsonomuto** [comment]
Yes, this is the best way to debug. Just use the javascript generated by the compiler.
